### PR TITLE
doc: Add issue templates and configure issue template chooser

### PR DESCRIPTION
**Pull request by ajaykarpur**
*Description of changes:*
- Adds more issue templates
- Configures the GitHub issue template chooser

*Testing done:*
N/A

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have used the regional endpoint when creating S3 and/or STS clients (if appropriate)
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this library
```
Comment (knakad) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
about: Suggest an improvement for this library
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question
+    url: https://forums.aws.amazon.com/
```
Comment (laurenyu) [inline .github/ISSUE_TEMPLATE/config.yml]: what's the reasoning behind redirecting to the AWS Forums for questions?

Comment (ajaykarpur) [reply .github/ISSUE_TEMPLATE/config.yml]: Updated to the `amazon-sagemaker` page on Stack Overflow.

**Comment by ajaykarpur** [comment]
> Does GH automagically check the ISSUE_TEMPLATE folder for templates?

I believe so: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository

**Pull request merged**